src/routes/chats.js

```diff
@@ -113,9 +113,13 @@ module.exports = (() => {
 	*
 	* @apiSuccess {Object[]} - List of messages.
 	* @apiSuccess {Number} -.id Message id.
-	* @apiSuccess {Number} -.users_id User ID of user who posted message.
+	* @apiSuccess {Number} -.users_id User ID.
 	* @apiSuccess {String} -.created Message timestamp with timezone; defaults to current.
 	* @apiSuccess {String} -.text Message text.
+	* @apiSuccess {Object[]} -.reactions List of emoji reactions.
+	* @apiSuccess {Object} -.reactions.- Emoji reaction.
+	* @apiSuccess {Number} -.reactions.-.users_id User id.
+	* @apiSuccess {String} -.reactions.-.emoji User emoji reaction.
```

Comment:
These were missing from the changes to the endpoint too; @apiSuccess is just meant to specify what each item in the example success response is.